solutions/usaco-831.mdx

```diff
@@ -67,26 +66,26 @@ public class TeamTicTacToe {
 	}
 
 	// Does a team win?
-	static int teamWins(char ch1, char ch2) {
-		// Check diagonals
+	static boolean teamWins(char ch1, char ch2) {
+		// Check diagonals like with a single cow
```

Comment:
wait what is this supposed to mean?

Reply:
like we did in the `cowWins` method

Reply:
oops i read the comment wrong.
```suggestion
		// Check diagonals with a single cow
```

Reply:
? how does that comment work?

Reply:
huh

Reply:
that code isn't checking a single cow?

Reply:
no it is, but the `like` is unnecessary?

Reply:
(optional though) ^^

Reply:
ok will change tomorrow

Reply:
i can do it